### DISCUSSION TITLE
widen the number of admins

### DISCUSSION THREAD

**Discussion by jgadsden** [maxGraph/maxGraph#13] [Development]
It would be good to have more admins for this project, @coclav is doing great, but we need more than 1

**Comment by coclav** [comment]
Please make yourself known here :-)

**Reply by mcyph** [reply]
Hey @coclav, we've been discussing at https://github.com/jsGraph/mxgraph/issues/8 whether we can create a separate "unstable"/refactoring branch where we can more easily collaborate on the source code together!

I'm not sure I'd need admin permissions to the main branch, but would it be possible to create a separate branch that @junsikshim, me and possibly others could contribute to based on the repo at https://github.com/mcyph/mxgraph/ ??

**Reply by jgadsden** [reply]
Hello @mcyph - I have created 'development' branch off the main branch. Will this do what you are asking for?

**Reply by mcyph** [reply]
@jgadsden thanks yeah I think so! I've opened a PR with my changes.

I'm not sure how this works exactly but it also could be an idea to allow a few of the people who are active to review+approve PR's prior to merging.

**Reply by jgadsden** [reply]
When you become a contributor then you get these permissions, but it needs an admin to make you a contributor (and I think it requires you to also have a PR, which you have done)

**Reply by jgadsden** [reply]
Hello @mcyph and @junsikshim - you have been active in submitting pull requests and reviews, I would like to widen ther admins to include you. Are you OK with this?

@coclav is this OK with you as well?

**Comment by tbouffard** [comment]
Based on https://github.com/maxGraph/maxGraph/issues/1#issuecomment-991838445 and as they are not contributing actively to this repository, I have just removed the `Admin` role of @jgadsden and the `Maintain` role @lreading. They now have no specific Role in this repository.

The current list of declared admin is
- @junsikshim 
- @mcyph 
- @tbouffard 

I guess @coclav have the maxGraph organization `Owner` role so he can also act as an `Admin` of this repository.

@coclav could you make me Admin or Owner of the maxGraph GitHub organization please? That way, we can add new repo, manage Team, manage secrets globally, .... without bothering you if you don't want or don't have time to contribute actively to the maxGraph project. Other could be added too. Thanks in advance
